src/MPoly.jl

```diff
@@ -808,7 +808,7 @@ function _evaluate(a, S, R, vars, vals::Vector{Rational{BigInt}})
 end
 
 function __evaluate(a, vars, vals, powers)
-   R = base_ring(a)
+   R = parent(a) isa MSeriesRing ? base_ring(a) : coefficient_ring(a)
```

Comment:
Apparently some `MSeriesRing` code calls `__evaluate`. Since I did not change for series, it now looks like this.